### PR TITLE
✨ Hebrew Support (希伯来文字实现)

### PR DESCRIPTION
**Pull request by yanivshaked**
Ni hao,

I have added xi bu lai (Hebrew) support.

Fei chang gan xie!,
Yaniv

### CODE REVIEW

File: lib/src/delegates/assets_picker_text_delegate.dart [hunk 1]
```diff
@@ -141,7 +141,7 @@ class EnglishTextDelegate implements AssetsPickerTextDelegate {
   String select = 'Select';
 
   @override
-  String unSupportedAssetType = 'Unsupported HEIC asset type.';
+  String unSupportedAssetType = 'Unsupported asset type.';
```
Comment (AlexV525) [inline lib/src/delegates/assets_picker_text_delegate.dart:144]: Currently, we can recognize all types of assets if they're recognized by the OS, so I suggest we keep this value and the field without making a default value breaking change (There will be some people that might suffer from this).

**Review by AlexV525 — APPROVED** [review]
LGTM overall, just a nit picking.

File: lib/src/delegates/assets_picker_text_delegate.dart (hunk 1 above)
Comment (yanivshaked) [reply lib/src/delegates/assets_picker_text_delegate.dart:144]: Ok, I will revert this change.

Comment (yanivshaked) [reply lib/src/delegates/assets_picker_text_delegate.dart]: Revert value changed for unSupportedAssetType (See last commit)

File: lib/src/delegates/assets_picker_text_delegate.dart [hunk 2]
```diff
@@ -154,3 +154,54 @@ class EnglishTextDelegate implements AssetsPickerTextDelegate {
...
+  String preview = 'תצוגה מקדימה';
+
+  @override
+  String select = 'בחר';
+
+  @override
+  String unSupportedAssetType = 'סוג קובץ אינו נתמך';
+
+  @override
+  String durationIndicatorBuilder(Duration duration) {
```
Comment (AlexV525) [inline lib/src/delegates/assets_picker_text_delegate.dart:198]: Sorry for not seeing this, but what do you think if we implement this as a default implementation in the abstract delegate?

Comment (AlexV525) [reply lib/src/delegates/assets_picker_text_delegate.dart:198]: After that we don't have to override this method in most of delegates.

Comment (yanivshaked) [reply lib/src/delegates/assets_picker_text_delegate.dart:198]: Correct! This will be much better. I'm on it.

Comment (yanivshaked) [reply lib/src/delegates/assets_picker_text_delegate.dart:198]: Apparently, this is not trivial in dart. Dart forces implementation of non abstract methods in child classes, and if I change from "implements" to "extends" I'm unable to override the String fields.
https://stackoverflow.com/questions/61738325/dart-forcing-me-to-implement-non-abstract-method-in-child-class
What do you think?

Comment (AlexV525) [reply lib/src/delegates/assets_picker_text_delegate.dart:198]: Maybe:
image
image

Comment (yanivshaked) [reply lib/src/delegates/assets_picker_text_delegate.dart:198]: Coo, static solves this issue. Implementing.

Comment (yanivshaked) [reply lib/src/delegates/assets_picker_text_delegate.dart]: See changes, I have also commited a fix to the Hebrew translation for unSupportedAssetType (as per the English change revert).

File: lib/src/delegates/assets_picker_text_delegate.dart [hunk 3]
```diff
@@ -144,13 +147,50 @@ class EnglishTextDelegate implements AssetsPickerTextDelegate {
...
+  @override
+  String select = 'בחר';
+
+  @override
+  String unSupportedAssetType = 'סוג קובץ HEIC אינו נתמך';
+
+  @override
+  String durationIndicatorBuilder(Duration duration) =>
+      AssetsPickerTextDelegate.defaultDurationIndicatorBuilder(duration);
+}
```
Comment (AlexV525) [inline lib/src/delegates/assets_picker_text_delegate.dart:196]:
```suggestion
}

```

**Review by AlexV525 — APPROVED** [review]
Only one thing left :)

**Comment by AlexV525** [comment]
Seems good to go, merging.

**Pull request merged**